src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/reducers/__tests__/chartBuilderReducer.test.ts

```diff
@@ -1071,5 +1152,74 @@ describe('chartBuilderReducer', () => {
         },
       });
     });
+
+    test('has correct state with initial configuration for a map', () => {
+      const initialConfiguration: Chart = {
+        legend: {
+          position: 'top',
+          items: [],
+        },
+        axes: {
+          major: {
+            type: 'major',
+            dataSets: [
+              {
+                indicator: 'indicator-1',
+                filters: ['filter-1'],
+              },
+            ],
+            referenceLines: [],
+          },
+        },
+        type: 'map',
+        height: 300,
+        title: 'Chart title',
+        alt: '',
+        boundaryLevel: 1,
+      };
+
+      const { result } = renderHook(() =>
+        useChartBuilderReducer({
+          data: [],
```

Comment:
We're currently not testing anything of value as we're not generating any `map.dataSetConfigs`.

Looks like we need to provide some data so that we can generate the initial map data set configs and assert that these are as expected.